backends/concrete-cpu/implementation/src/implementation/wop_simulation.rs

```diff
@@ -75,8 +72,7 @@ pub fn extract_bits(
     ciphertext_modulus_log: u32,
     security_level: u64,
 ) {
-    let sw_csprng = &mut SoftwareRandomGenerator::new(Seed(0));
-    let mut csprng = to_generic(sw_csprng);
+    let mut csprng = RandomGenerator::<SoftwareRandomGenerator>::new(Seed(0));
```

Comment:
should be a parameter ?

Reply:
yes that could be as parameter to be able to have repeatable simulation, but it will imply a lot of more change and is out of the scope of this PR.